packages/DataGrid/src/components/columns/renderColumnExpand.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Maximize from "@paprika/icon/lib/Maximize";
+import ColumnDefinition from "../ColumnDefinition";
+
+const propTypes = {
+  attrs: PropTypes.shape({}),
+  isActiveCell: PropTypes.bool,
+  isActiveRow: PropTypes.bool,
+};
+
+const defaultProps = {
+  attrs: {},
+  isActiveCell: false,
+  isActiveRow: false,
+};
+
+function Expand(props) {
+  const { attrs, isActiveCell, isActiveRow } = props;
+  const [opacity, setOpacity] = React.useState(0);
+
+  function show() {
+    setOpacity(() => 1);
+  }
+
+  function hide() {
+    setOpacity(() => 0);
+  }
+
+  return (
+    <div
+      style={{ opacity: opacity || isActiveCell || isActiveRow ? 1 : 0 }}
+      css={`
+        cursor: pointer;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      `}
+      onMouseOver={show}
+      onMouseLeave={hide}
+      onFocus={show}
+      onBlur={hide}
+      {...attrs}
+    >
+      <div>
+        <Maximize />
+      </div>
+    </div>
+  );
+}
+
+Expand.propTypes = propTypes;
+Expand.defaultProps = defaultProps;
+
+export default function renderColumnExpand(options = {}) {
```

Comment:
The problem I'm facing here is to create reusable columns, I tried to encapsulate this as a component but the problem of doing that, its that will end creating a new component and making impossible for `DataGrid.js` to access the **props** on ColumnDefinition, 
Returning a component instead of instancing it will give me access to all props on Column definition that I can use on DataGrid file like `Column.props.cell`, `Column.props.width` etc. 

To me, this is similar when you create a `renderSomething` function inside your code.

If you can think of a better idea about how to achieve this, I can modify it.